IrinaLoginovaHomeWork_3_1.cpp

```diff
@@ -0,0 +1,17 @@
+#include <cstdint>
+#include <cstring>
+#include <iostream>
+
+int main() {
+  constexpr int n = {150};
```

Comment:
```suggestion
  constexpr int size_string {150};
```
просто `n` не очень читабельное в контексте применения длины строки

Reply:
Поменяла на SizeString. Просто в таком формате написаны все переменные.